sigstore/_verify.py

```diff
@@ -236,27 +246,91 @@ def verify(
 
         logger.debug("Successfully verified signature...")
 
-        # Retrieve the relevant Rekor entry to verify the inclusion proof and SET
-        entry = self._rekor.log.entries.retrieve.post(
-            signature.decode(),
-            sha256_artifact_hash,
-            base64.b64encode(certificate).decode(),
-        )
-        if entry is None:
-            return RekorEntryMissing(
-                signature=signature.decode(), sha256_artifact_hash=sha256_artifact_hash
+        entry: Optional[RekorEntry]
+        if offline_rekor_entry is not None:
+            # TODO(ww): This should all go in a separate API, probably under the
+            # RekorEntry class.
+            logger.debug("offline Rekor entry: checking consistency")
```

Comment:
nit: Remove debug statement? Not sure if you want to keep it or not (if so, change "consistency")

Reply:
Yeah, bad word choice on my part (I'm checking consistency between the entry and the signing artifacts, not consistency with the log). I'll reword.